cmd/rollkit/commands/run_node.go

```diff
@@ -193,17 +187,14 @@ func addNodeFlags(cmd *cobra.Command) {
 	rollconf.AddFlags(cmd)
 }
 
-// startMockGRPCServ starts a mock gRPC server for the dummy DA
-func startMockGRPCServ() *grpc.Server {
-	srv := proxy.NewServer(goDATest.NewDummyDA(), grpc.Creds(insecure.NewCredentials()))
-	lis, err := net.Listen("tcp", "127.0.0.1"+":"+strconv.Itoa(7980))
+// startMockDAServJSONRPC starts a mock JSONRPC server
+func startMockDAServJSONRPC(ctx context.Context) *proxy.Server {
+	addr, _ := url.Parse(rollkitConfig.DAAddress)
```

Comment:
Handle error (panic?) or comment that this function is only called when `DAAddress` is not changed - it has default value, so it's assumed it's always correct.

Reply:
returning err now